fieldservice/models/res_config_settings.py

```diff
@@ -63,6 +65,24 @@ class ResConfigSettings(models.TransientModel):
         related='company_id.auto_populate_persons_on_location',
         readonly=False)
 
+    def get_values(self):
+        res = super(ResConfigSettings, self).get_values()
+        res.update(auto_populate_the_equipments=self.env['ir.config_parameter'].sudo().
```

Comment:
@murtuzasaleh LGTM can we just get this line shortened? Travis build still has red light